### ISSUE TITLE
Mesos monitoring

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#4881]
We are willing to support monitoring ***[Mesos](http://mesos.apache.org/)*** with netdata.

To get started, we need a user that will act as a [sponsor](https://docs.netdata.cloud/contributing/#sponsor-a-part-of-netdata) during the implementation.
He/she will assist us with the following:

  - Specify the metrics required and how they should be presented in charts.
  - Specify the alarms that would make sense for each metric.
  - When the implementation passes QA, test the implementation in production.
  - Use the charts and alarms in their day to day work and provide additional feedback after the collector is delivered.

We would also appreciate getting votes :+1: for this issue from the community, to understand the interest in the particular collector.

**Comment by rob-duewer** [comment]
So while waiting on "real" mesos/marathon support (and by all means, keeping working on Kube first; I'm sure that has a bigger audience), I quickly threw together this extension to cgroup-name.sh (https://github.com/netdata/netdata/blob/master/collectors/cgroups.plugin/cgroup-name.sh.in) I thought I'd post it up here in case anyone was interested.  I figured that the plan to do true support will likely completely supersede these efforts, so I didn't bother to kick it in as a PR.  But since I've found it useful enough in the interim, for our use case at least, that I thought it might others might want to try it out.  In this case, it renames the presented counter values for mesos managed docker containers to something that is more useful to identify them, in coordination with the orchestration model.

In this case, it identifies containers of this type using the current "mesos-" naming convention of the container that is instanced.  When found (and using the docker classic model), it retrieves the image name and tag, crunches that down to eliminate repo/library information, replaces the characters that would make the result not play well with the UI and other backends (we use graphite/grafana, with which netdata's integration is really great!), and adds on the last portion of the UUID to keep it unique and easy to find in the mesos UI.  

I figured it's not useful to everyone, nor is particularly hardened.  And it's not as good as getting a list of the current marathon app ids, chasing those down to the mesos spawned containers UUIDs (which I suspect will be the ideal use case when this ticket gets formally considered). But since it has no reliance on anything except what is visible to the local cotainerizer, it was a nice easy hack for the time being. I certainly find it useful so thought I would post it here in case it was interesting to anyone else while this ticket for more complete support is pending consideration.

Keep up the great work guys!
[cgroup-name.sh.txt](https://github.com/netdata/netdata/files/2979941/cgroup-name.sh.txt)
